app/models/date_time_field_type.rb

```diff
@@ -42,7 +42,7 @@ def mapping_field_name
   end
 
   def timestamp_is_valid?
-    if @timestamp.nil?
+    if @timestamp.blank? || @timestamp.nil?
```

Comment:
Actually `blank?` appears to not work on a nil object. Instead I've cast the object to a consistent String (`nil.to_s == ""`) and run `#empty?` to confirm this